homeassistant/components/nissan_leaf/binary_sensor.py

```diff
@@ -0,0 +1,44 @@
+"""Plugged In Status Support for the Nissan Leaf."""
+import logging
+
+from homeassistant.components.nissan_leaf import (
+    DATA_LEAF, DATA_PLUGGED_IN, LeafEntity)
+
+_LOGGER = logging.getLogger(__name__)
+
+DEPENDENCIES = ['nissan_leaf']
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up of a Nissan Leaf binary sensor."""
+    _LOGGER.debug(
+        "binary_sensor setup_platform, discovery_info=%s", discovery_info)
+
+    devices = []
+    for key, value in hass.data[DATA_LEAF].items():
```

Comment:
Please use more descriptive variable names than `key` and `value`. What are these variables?

Reply:
Fixed with commit https://github.com/filcole/home-assistant/commit/648929cb56df72a0a34fbfd550465f934f4fb6dc